### ISSUE TITLE
getting ads on an adblocker...

### ISSUE THREAD

**Issue by IcyChillzzz** [0x48piraj/fadblock#71]
very interesting that I get ads from **an ad blocker** every video to purchase the lifetime license.. when the whole point of the adblocker is blocking ads

**Comment by SonicHax** [comment]
agreed, and its intrusive to the point i'm thinking about uninstalling it and dealing with the youtube ads, don't know which one is worse at this point

**Comment by gotexis** [comment]
use the userscript in https://github.com/0x48piraj/fadblock/blob/master/src/safari/js/fadblock.user.js

**Comment by 0x48piraj** [comment]
@IcyChillzzz and @SonicHax, I understand your issue and the experience is similar, I messed up a bit, the initial popup comes after ~500 ads which I thought was a good enough time where it'll be okay to show the popup with high frequency, which it is not.

I have already significantly reduced the frequency of the popup (by ~1533%) and it'll take the dev team a day or two to give it the green light and roll out the new version.

Again, @gotexis, I see you copt-pasting this over every thread, please read the announcement.

> I genuinely hope you all can appreciate the immense amount of time and effort it requires to sustain a project like this all the while YouTube keeps changing the rules, trying to shut-out everything. All this coupled with working >80 hours/week on my own thing (not FadBlock). If you're not satisfied with the decision where you have to pay a one-time contribution as a gesture for "job well done and keep at it", you still have the option to manually roll back to version 1.3. I won't remove it; it just won't receive further updates. I believe that's a fair compromise.

It's not a hidden thing, the problem is, 1.3 doesn't work with the new YouTube patches but these patches haven't gone global so it may or may not work. If it does and you don't want to wait till I fix the frequency problem, you're free to downgrade manually.

**Comment by 0x48piraj** [comment]
@timcortesi, I was also not looking to maintain this project or actively invest upwards of 30 hours a week on this.

The responsibilities changed quite a bit in the last few weeks. I can't say if I would have continued investing this significant time if it weren't for the donations and the positive notes from people. I thank you for it. I have adjusted the popup frequency and am happy to throttle it down some more if it's still not up to the mark but without asking actively for donations, I have noticed you don't get any, at all.
